### PR TITLE
feat: quests feature toggle

### PR DESCRIPTION
**Pull request by AjimenezDCL**
https://app.zenhub.com/workspaces/unity-6047bad476c3c0001942ee91/issues/decentraland/unity-renderer/455

The old `QUEST_ENABLED` flag in the URL is deprecated and the status of the feature will be retrieved through the Unleash feature toggle: https://features.decentraland.systems/#/features/strategies/explorer-quests_enabled

### CODE REVIEW

File: kernel/packages/shared/meta/types.ts [hunk 1]
```diff
@@ -63,5 +63,6 @@ export type CommsConfig = {
 
 export enum FeatureFlags {
   WEARABLES_V2 = 'wearables_v2',
+  QUESTS_ENABLED = 'quests_enabled',
```
Comment (pbosio) [inline kernel/packages/shared/meta/types.ts]: `QUESTS_ENABLED` the url param flag like this would be `ENABLE_QUESTS_ENABLED` 😅

Comment (menduz) [reply kernel/packages/shared/meta/types.ts]: Pato has a point, probably adding the `enabled` to a feature flag may be an anti-naming-pattern `quests` may suffice

Comment (AjimenezDCL) [reply kernel/packages/shared/meta/types.ts]: Sure, gonna change it

**Review by menduz — APPROVED** [review]

**Comment by menduz** [comment]
Should we merge this?

**Comment by AjimenezDCL** [comment]
We are waiting for the release to be done

**Pull request merged**